DependencyInjection/Compiler/CacheSchemaSubscriberPass.php

```diff
@@ -19,26 +20,34 @@ class CacheSchemaSubscriberPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        $subscriberId = 'doctrine.orm.listeners.pdo_cache_adapter_doctrine_schema_subscriber';
+        // available in Symfony 5.4 and higher
+        $this->injectAdapters($container, 'doctrine.orm.listeners.doctrine_dbal_cache_adapter_schema_subscriber', DoctrineDbalAdapter::class);
```

Comment:
you can suppress this psalm violation with inline phpdoc

Reply:
Indeed. That's done.